### ISSUE TITLE
.applyBehavior vs .applyBehaviour (UK/US spelling) : zoom doesn't work

### ISSUE THREAD

**Issue by wahlatlas** [floledermann/mapmap.js#5]
I can't get the zoom to work on the Austria map example. While I don't fully grasp what is going on, I am a bit puzzled to see the example featuring this line (US spelling)

```
.applyBehavior(mapmap.behavior.zoom())
```

which is in line with D3's behavior.zoom. But then in _mapmap.js_ I find some UK spelling like this

```
mapmap.prototype.applyBehaviour = function(spec, selection) {
```

maybe this is very well thought through and the problem lies elsewhere but it could literally spell trouble

**Comment by floledermann** [comment]
Wow, this is really bad :) I changed method names to US spelling before the public release, but obviously overlooked `applyBehaviour` - will be fixed, all indentifiers should use US spelling.

**Comment by floledermann** [comment]
Fixed in 0.2.3
